code/controllers/subsystem/non_firing/titlescreen.dm

```diff
@@ -3,32 +3,15 @@ SUBSYSTEM_DEF(title)
 	flags = SS_NO_FIRE
 	init_order = INIT_ORDER_TITLE
 
-/datum/controller/subsystem/title/Initialize()
-	var/list/provisional_title_screens = flist("config/title_screens/images/")
-	var/list/title_screens = list()
-	var/use_rare_screens = prob(1)
-
-	for(var/S in provisional_title_screens)
-		var/list/L = splittext(S,"+")
-		if(L.len == 1 && L[1] != "blank.png")
-			title_screens += S
-
-		else if(L.len > 1)
-			if(use_rare_screens && lowertext(L[1]) == "rare")
-				title_screens += S
 
-	if(!isemptylist(title_screens))
-		if(length(title_screens) > 1)
-			for(var/S in title_screens)
-				var/list/L = splittext(S,".")
-				if(L.len != 2 || L[1] != "default")
-					continue
-				title_screens -= S
-				break
-
-		var/file_path = "config/title_screens/images/[pick(title_screens)]"
+/datum/controller/subsystem/title/Initialize()
+	var/file_path = "config/title_screens/images/screens.dmi"
+	var/icon/icon = new(fcopy_rsc(file_path))
+	var/splashscreen = "[rand(1,2)]"
 
-		var/icon/icon = new(fcopy_rsc(file_path))
+	for(var/turf/simulated/wall/indestructible/splashscreen/splash in world)
+		splash.icon = icon
+		splash.icon_state = splashscreen
+		if(splashscreen == "1" || splashscreen == "2")
+			splash.name = "by Reshig"
```

Comment:
This isn't very maintainable. If we really want to store author name, we need to do something better like put them into a list, or make datum subtypes for them. Then pick a random one and assign author name dynamically.
```dm
var/list/splashscreens = list(
     list("icon" = 'myiconpath', "icon_state" = "iconname", "author" = "Reshig"),
     list(...),
)
```